sos/plugins/openstack_horizon.py

```diff
@@ -26,7 +26,10 @@ class OpenStackHorizon(Plugin):
 
     plugin_name = "openstack_horizon"
     profiles = ('openstack', 'openstack_controller')
-    option_list = []
+
+    option_list = [
+        ("exclude", "Ignore files matching this path spec", "", "")
+    ]
```

Comment:
If you find yourself adding the same option to more than one or two plugins, that is a sign that either something is not right, or that that option should be a [global option](https://github.com/sosreport/sos/wiki/Plugin-options).

In this case I don't think this is something that we want to make into a global option.